catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -27,6 +27,11 @@
                 default="60"
         />
         <AD
+                description="Maximum time to wait before starting a standing query (seconds)"
```

Comment:
Since the default is fairly long at 60 minutes, would it make more sense for an admin if this parameter were treated in minutes?